<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComputeTurnoverDialog</class>
 <widget class="QDialog" name="ComputeTurnoverDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calcul du chiffre d'affaires</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCompute">
       <property name="text">
        <string>Calculer</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons.qrc">
         <normaloff>:/icons/img/rate.png</normaloff>:/icons/img/rate.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>291</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Période</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="3">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Fin</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="dtBeginPeriod">
          <property name="time">
           <time>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
           </time>
          </property>
          <property name="currentSection">
           <enum>QDateTimeEdit::DaySection</enum>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
          <property name="timeSpec">
           <enum>Qt::LocalTime</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Début</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QDateEdit" name="dtEndPeriod">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lbBillingNb">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>CA</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leTurnover">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>€ (euros)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lbCompute">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCompute</sender>
   <signal>released()</signal>
   <receiver>ComputeTurnoverDialog</receiver>
   <slot>computeTurnover()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>computeTurnover()</slot>
  <slot>endDateControl(QDate)</slot>
  <slot>beginDateControl(QDate)</slot>
 </slots>
</ui>
